cfn.yaml

```diff
@@ -101,10 +101,10 @@ Resources:
       Runtime: java8
       Timeout: 60
 
-  Lambda:
+  LiveAppVersionsLambda:
     Type: AWS::Lambda::Function
     Properties:
-      FunctionName: !Sub live-app-versions-${Stage}
+      FunctionName: !Sub live-app-versions-v2-${Stage}
```

Comment:
Renaming due to https://github.com/guardian/live-app-versions/pull/11#discussion_r413780846